lib/parse-imports.js

```diff
@@ -1,6 +1,22 @@
 var valueParser = require("postcss-value-parser")
 var stringify = valueParser.stringify
 
+function getMedia(params) {
+  var media = []
+  var last = params.reduce(function(medium, node, index) {
+    if (index < 2) {
+      return ""
+    }
+    if (node.type === "div" && node.value === ",") {
```

Comment:
div is divider (',', '/', ':')
This function just split params with comma divider after second (url+space) node